notificationdemo/source/java/src/org/kaaproject/kaa/demo/notification/NotificationDemo.java

```diff
@@ -19,30 +19,40 @@
 import org.kaaproject.kaa.client.DesktopKaaPlatformContext;
 import org.kaaproject.kaa.client.Kaa;
 import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.SimpleKaaClientStateListener;
 import org.kaaproject.kaa.client.notification.NotificationListener;
 import org.kaaproject.kaa.client.notification.NotificationTopicListListener;
 import org.kaaproject.kaa.client.notification.UnavailableTopicException;
 import org.kaaproject.kaa.common.endpoint.gen.SubscriptionType;
 import org.kaaproject.kaa.common.endpoint.gen.Topic;
-
+import org.kaaproject.kaa.schema.sample.notification.SecurityAlert;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Scanner;
 
 /**
  * A demo application that shows how to use the Kaa notifications API.
  */
 public class NotificationDemo {
+
     private static final Logger LOG = LoggerFactory.getLogger(NotificationDemo.class);
     private static KaaClient kaaClient;
+    /**
+     * The list of all available notification topics.
+     */
+    private static List<Topic> topics;
+    /**
+     * Topics client subscribed
+     */
+    private static List<Topic> subscribedTopics;
 
     public static void main(String[] args) {
         LOG.info("Notification demo started");
-        LOG.info("--= Press any key to exit =--");
-        kaaClient = Kaa.newClient(new DesktopKaaPlatformContext(), null, true);
+
+        kaaClient = Kaa.newClient(new DesktopKaaPlatformContext(), new SimpleKaaClientStateListener(), true);
 
         // A listener that listens to the notification topic list updates.
```

Comment:
Please, replace all `// ...` comments with `/* ... */` ones to keep same comment style.